### PR TITLE
Update top bar

### PR DESCRIPTION
**Pull request by kaytcampbell**
For the super long foundation_and_overrides file, you can pretty much ignore everything up till line 1397 which is where the only changes to the file are. The rest is just default foundation code.

The rest of this PR is to build and style the top-bar and side-bar.

### CODE REVIEW

**Comment by adamrmelnyk** [comment]
:book:

File: app/assets/stylesheets/ama_layout/layout/sidebar.scss [hunk 1]
```diff
@@ -1,104 +1,108 @@
...
+
+   a{
+     color: white;
+     max-height: inherit;
     }
-    span.nav-icon {
-     color:white;
+
+    span.nav-icon{
+     color: white;
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout/sidebar.scss]: Should be using $white var

File: app/assets/stylesheets/ama_layout/layout/sidebar.scss [hunk 2]
```diff
@@ -1,104 +1,108 @@
...
   max-height: 0;
   -webkit-transition: max-height 0.15s ease-out;
   -moz-transition: max-height 0.15s ease-out;
   transition: max-height 0.15s ease-out;
   margin-left: $base-margin*12.4;
   margin-top: -$base-margin*3;
+
   li{
-    color:white;
+    color: white;
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout/sidebar.scss]: Should be using $white var

File: app/views/ama_layout/_siteheader.html.erb [hunk 3]
```diff
@@ -1,17 +1,40 @@
...
+       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
+      <li class="toggle-topbar menu-icon"><a href="#"><span></span></a></li>
+    </ul>
+    <section class="top-bar-section show-for-small-only">
+      <ul class="right">
+        <%= render partial: "ama_layout/main_top_nav_item", collection: navigation, as: :nav_item if navigation %>
+        <li class="divider"></li>
+        <li class="show-for-small-only"><a href="#">Help</a></li>
+        <li class="divider show-for-small-only"></li>
+        <li class="show-for-small-only"><a href="#">Contact Us</a></li>
```
Comment (kathleenAMA) [inline app/views/ama_layout/_siteheader.html.erb]: Will we be adding in actual links to here and Help?

**Comment by kathleenAMA** [comment]
:hankey: Just those two white variables and it should be good to go.

File: app/assets/stylesheets/ama_layout/layout/sidebar.scss [hunk 4]
```diff
@@ -1,104 +1,108 @@
...
+  color: $charcoal;
   text-transform: uppercase;
-  background:white;
-  padding: $base-margin*3;
+  background: $white;
+  padding: $base-padding*3;
 }
 
-.dashboard-nav {
+.dashboard-nav{
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout/sidebar.scss]: If we're following Github's style guide: http://primercss.io/guidelines/#spacing, we should keep the space before `{`.

File: app/views/ama_layout/_siteheader.html.erb [hunk 5]
```diff
@@ -1,17 +1,40 @@
...
+        <li class="divider show-for-small-only"></li>
+        <% if current_user %>
+        <li><a href="/logout">Sign Out</a></li>
+        <% end %>
+      </ul>
+    </section>
+    <section class="top-bar-section show-for-medium-up">
+      <ul class="right">
+        <li><a href="#">Help</a></li>
```
Comment (MathieuGilbert) [inline app/views/ama_layout/_siteheader.html.erb]: Need real links here too.

**Comment by MathieuGilbert** [comment]
:poop: Need a gem version bump as well.

File: app/assets/stylesheets/ama_layout/layout/sidebar.scss (hunk 4 above)
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout/sidebar.scss]: @MathieuGilbert Just FYI, Kayt and I decided that as part of our code standard we don't want a space there.

**Comment by mvandenbeuken** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Comment by mvandenbeuken** [comment]
:wave: @kathleenAMA :boom: !!!

How does the formatting look?

**Comment by MathieuGilbert** [comment]
If we're adding exceptions to our style rules, please update the relevant section here: https://web-dev-wiki-pv01.corp.ads/docuwiki/doku.php?id=rules

**Comment by kathleenAMA** [comment]
@mvandenbeuken  :art: :art: :art: !

**Pull request merged**